ApplicationLibCode/Application/Tools/RiaLogging.cpp

```diff
@@ -163,6 +163,35 @@ void RiaDefaultConsoleLogger::writeToConsole( const std::string& str )
 #endif
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RiaLogging::setLastMessage( const QString& message )
+{
+#pragma omp critical( critical_section_logging )
+    sm_lastMessage     = message;
+    sm_lastMessageTime = std::chrono::high_resolution_clock::now();
+}
+
```

Comment:
Do you need curly braces to make the `pragma` apply to both statements?

Reply:
I think we do: https://stackoverflow.com/a/59904904/3608

Reply:
Fixed